NuGet.config

```diff
@@ -3,10 +3,12 @@
   <packageSources>
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
-    <add key="darc-pub-dotnet-core-setup-f375e03" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-f375e037/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-core-setup-f96e1b9" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-f96e1b90/nuget/v3/index.json" />
     <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
     <add key="dotnet3" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3/nuget/v3/index.json" />
     <add key="dotnet3-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3-transport/nuget/v3/index.json" />
```

Comment:
I don't understand why the dotnet 3.0 feeds are here, in aspnetcore-tooling, and in ef6. But, I feel safer leaving them alone. (aspnetcore and efcore only have the 3.1 feeds.)

Thoughts❔

Reply:
Maybe there are some 3.0 packages (templates maybe?) that are pulled in. You can probably just remove though.

Reply:
Hmm, good point. Might be old stuff like System.Memory and System.Net.Http. Those are mentioned near the bottom of eng/Versions.props (now that I look; had only checked eng/Version.Details.xml). The missing bits _might_ be available in dotnet-core (e.g. aspnetcore gets System.Net.Http v4.3.2 w/o the 3.0 feeds) but I'd rather do that experiment in 3.1.9.

Reply:
Wouldn't those be released at this point on nuget.org?

Reply:
Probably. My hesitation is about experimenting w/ something we see in three different repos when removing the 2 relatively small feeds is at best nice-to-have. Then, again, I might as well give it a shot because agent wait times have dropped significantly. Coming up&hellip;